nexus/src/app/background/tasks/support_bundle_collector.rs

```diff
@@ -605,16 +606,16 @@ impl BundleCollection {
                 "Streaming bundle chunk";
                 "bundle" => %self.bundle.id,
                 "offset" => offset,
-                "length" => remaining,
+                "length" => chunk_size,
             );
 
             sled_client.support_bundle_transfer(
                 &zpool, &dataset, &support_bundle, offset, body
             ).await.with_context(|| {
-                format!("Failed to transfer bundle: {remaining}@{offset} of {total_len} to sled")
+                format!("Failed to transfer bundle: {chunk_size}@{offset} of {total_len} to sled")
             })?;
 
-            offset += self.transfer_chunk_size.get();
+            offset += chunk_size;
```

Comment:
This technically wasn't "wrong" before, because we didn't use the offset after the very last call, but over-shooting the actual length of the support bundle still felt fishy, so I changed this.

(The resulting behavior should be identical)